index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Addisalem seifu Portofolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <p>
+                My Logo
+            </p>
+            <img src="./img/menue.png" alt="">
+        </nav>
+    </header>
+    <section>
+        <div class="details-section">
+            <h1>I'm Addisalem am Very Happy to see you</h1>
+            <p class="details">i am a software developer by profesion and i help develop many softwares, you can go ahed and check out some of my projects in my portofolio, both mobile and web based and continuing to do so. if you need help regarding software development or have a project in mind feel free to contact me.</p>
+            <a href="#"><h2>LET'S CONNECT</h2></a>
+        </div>
+        <ul  class="pic-container">
+                <li><a href="https://twitter.com/beidework"><img src="./img/twitter.png" alt="twitter icon"></a></li>
+                <li><a href="https://www.linkedin.com/in/addisalemseifu/"><img src="./img/linkedin.png" alt="linkedin icon"></a></li>
+                <li><a href="https://github.com/addisalemseifu"><img src="./img/github.png" alt="github icon"></a></li>
+        </ul>
```

Comment:
The icons still now showing as the original design
### Original design
![right](https://user-images.githubusercontent.com/58376020/125995429-fd6c2c40-af69-4587-b9b9-90027def20c7.png)
### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/125995453-1e800f91-b017-4c35-a0db-e2517812007c.png)